flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/FlinkKubeClient.java

```diff
@@ -104,6 +106,67 @@ KubernetesWatch watchPodsAndDoCallback(
 		Map<String, String> labels,
 		WatchCallbackHandler<KubernetesPod> podCallbackHandler);
 
+	/**
+	 * Create the ConfigMap with specified content. If the ConfigMap already exists, a FlinkRuntimeException will be
```

Comment:
Might make sense to introduce some kind of `KubernetesException`, rather than throwing `FlinkRuntimeException`.

Reply:
Make sense. I will introduce a `KubernetesException`.